spec/core/chain/schemas/ChainInfoDTO.yml

```diff
@@ -0,0 +1,18 @@
+type: object
+required:
+  - height
+  - finalizedHeight
```

Comment:
is `finalizedHeight` duplicated value? User can get it from `latestFinalizedBlock.height`

Reply:
@Vektrat